### PR TITLE
revision for clarity

### PR DESCRIPTION
**Pull request by garnerlogan65**
reorg for clarity

### CODE REVIEW

File: content/bootloader.md [hunk 1]
```diff
@@ -17,13 +17,13 @@ section: software-troubleshooting
...
 
 ### Important Note
 
-If you need to configure grub-pc (for example, after an update), installing grub to all devices will break GRUB. You will need to install to `/dev/sda` _not_ `/dev/sda1`.
+If you need to configure grub-pc (for example, after an update), installing GRUB to all devices will break GRUB. You will need to install to `/dev/sda` _not_ `/dev/sda1`.
 
-On a fresh install of Pop!_OS 18.04, <u>systemd-boot</u> is used rather than the <u>GRUB</u> boot-loader, and the following instructions do not apply please refer to the <u>systemd-boot</u> section on this page.
+On a fresh install of Pop!_OS 18.04 and newer, <u>systemd-boot</u> is used rather than the <u>GRUB</u> bootloader, and the following instructions do not apply please refer to the <u>systemd-boot</u> section on this page.
```
Comment (cwsystem76) [inline content/bootloader.md:26]: This is a run-on sentence. Needs to be something like "the following instructions do not apply; please refer to" etc, or two separate sentences.

This is also one of the pages that could _really_ benefit from having some sections collapsed by default.

**Comment by garnerlogan65** [comment]
I'll  have to update this. I want collapsable sections for pop and ubuntu

**Comment by garnerlogan65** [comment]
or rather EFI/legacy

File: content/bootloader.md (hunk 1 above)
Comment (gangwerz) [reply content/bootloader.md]: I believe I addressed this concern with my update

**Review by ahoneybun — APPROVED** [review]
LGTM I like these changes.

**Review by n3m0-22 — APPROVED** [review]
This looks good to me.

**Pull request merged**